README.md

```diff
@@ -12,38 +12,13 @@ Follow these steps to run bookmark locally:
 this is not suitable for bookmark as we need them running side by side. One way to do this
 is to take [this](https://raw.githubusercontent.com/ArchiveBox/ArchiveBox/dev/bin/setup.sh)
 script and strip the `docker` and `docker-compose` parts.
-* In the `priv/static` folder create the archive folder `mkdir -p priv/static/archive`,
-`cd` into it and initialize the archive: `archivebox init --setup`.
-* Edit `ArchiveBox.conf` with the following settings
-
-```
-[SERVER_CONFIG]
-SECRET_KEY = <your secret key>
-TIMEOUT = 120
-SAVE_TITLE = TRUE
-SAVE_WGET = TRUE
-SAVE_WARC = FALSE
-SAVE_FAVICON = FALSE
-SAFE_PDF = TRUE
-SAVE_SCREENSHOT = TRUE
-SAVE_DOM = FALSE
-SAVE_SINGLEFILE = FALSE
-SAVE_READABILITY = FALSE
-SAVE_GIT = FALSE
-SAVE_MEDIA = FALSE
-SUBMIT_ARCHIVE_DOT_ORG = FALSE
-CHECK_SSL_VALIDITY = FALSE
-
-[ARCHIVE_METHOD_TOGGLES]
-SAVE_MERCURY = FALSE
-```
-
+* Setup archivebox running `./setup-archivebox.sh`
 * Make sure you have postgres running and `config/dev.exs` is setup to
 talk to your db. 
 * Install elixir dependencies with `mix deps.get`.
 * Create and migrate your database with `mix ecto.setup`.
 * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`.
 * For anonymous archives to work, you must register a new user with `anonymous@bookmark.org`
-as email.
+as email, it can be done running the seed: `run priv/repo/seeds.exs`.
```

Comment:
I'm not sure what `run` is. Is this a mix command?

Reply:
It's missing the `mix`

Reply:
Let's add it